src/generators/epoch.js

```diff
@@ -19,6 +26,8 @@ exports.generateEpoch = ({ transactionCount, blockCount, totalFeesPaid, totalOut
     let feesPaid = 0;
     let output = 0;
     const txsInBlock = isLastBlock ? totalTxs : randomNumber(0, totalTxs);
+    let utxosInBlock = isLastBlock ? totalUtxos : randomNumber(0, txsInBlock);
+    if (totalUtxos - utxosInBlock < 0) utxosInBlock = totalUtxos;
```

Comment:
Isn't it the same to do `let utxosInBlock = isLastBlock ? totalUtxos : randomNumber(0, totalUtxos);` instead?

```suggestion
    let utxosInBlock = isLastBlock ? totalUtxos : randomNumber(0, totalUtxos);
```